automation/taskcluster/decision_task_nightly.py

```diff
@@ -53,7 +53,9 @@ def generate_build_task(apks, is_staging):
             checkout +
             ' && python automation/taskcluster/helper/get-secret.py'
             ' -s {} -k dsn -f .sentry_token'.format(sentry_secret) +
-            ' && ./gradlew --no-daemon -PcrashReports=true clean test assembleRelease'),
+            ' && python automation/taskcluster/adjust/get-secret.py'
+            ' -s project/mobile/fenix/adjust -k Greenfield -f .adjust_token' +
```

Comment:
Should there be different "adjust" tokens for staging as for regular nightlies? FWIW, we use different sentry tokens for staging than for nightlies

Reply:
Also, I don't think you need the `+` unless it's after a `.format(...)` here :)

Reply:
I need to figure out how to get this to work for all four flavors: greenfield, firefoxNightly, firefoxBeta, and firefoxRelease. Could we separate the release builds into four steps with different keys named after the flavor?

Reply:
I think I'm missing some context here:
* What is "greenfield" in comparison to nightly? Is it a new release channel?
* Are we releasing Fenix in a "beta" and "release" flavour soon? (why are they named "firefox$flavour" rather than "fenix$flavour"?)

Thanks :)

Reply:
Greenfield is org.mozilla.fenix without any changes to make it replace Fennec, such as the App ID, shared user ID, etc. We're keeping a greenfield flavor around in case anyone changes their minds about replacing Fennec right away. We can't release the Fennec variants on the Play Store except to replace Fennec.

Reply:
I don't quite understand what you mean by "Greenfield is [Fenix] without any changes to make it replace Fennec" - isn't Fenix significantly different from Fennec already? What Fenix changes are being omitted for the Greenfield flavour?

Reply:
So, Greenfield has a new app id and no Fennec compatibility changes. It can deployed as a standalone beta on Google Play. The Firefox* variants have the same app id and can only be deployed AS Fennec under the same Google Play Store entries used for Fennec.